src/validation.cpp

```diff
@@ -2218,11 +2213,11 @@ static void UpdateTip(CTxMemPool& mempool, const CBlockIndex* pindexNew, const C
     }
 
     bilingual_str warning_messages;
-    if (!active_chainstate.IsInitialBlockDownload()) {
+    if (!this->IsInitialBlockDownload()) {
```

Comment:
There's no need to use `this->` to call member functions. `this->` is implicitly added before the names: https://en.cppreference.com/w/cpp/language/this.

Reply:
I prefer the `this->` prefix to be explicit about a member function call; last time I talked about this with others it was uncontroversial.

Reply:
Curious. I've never come across this style before.